docs/blocks-storage/store-gateway.md

```diff
@@ -814,7 +814,9 @@ blocks_storage:
     # CLI flag: -blocks-storage.tsdb.head-compaction-concurrency
     [head_compaction_concurrency: <int> | default = 5]
 
-    # If TSDB head is idle for this duration, it is compacted. 0 means disabled.
+    # If TSDB head is idle for this duration, it is compacted. Note that up to
+    # 25% jitter is added to the value to avoid ingesters compacting
```

Comment:
I am just wondering, what is the reason to leave the jitter percentage unconfigurable  to the user?

Reply:
We didn't have a strong opinion here, other than following how jitter has typically been added in other places.

I don't think there would be opposition to be being configurable though.

Reply:
I see.  Make sense. I agree we can make it configurable when the need arises in a separate PR.